modules/update-upgrading-web.adoc

```diff
@@ -1,17 +1,22 @@
 // Module included in the following assemblies:
 //
 // * updating/updating-cluster.adoc
+// * updating/updating-cluster-between-minor.adoc
+
+ifeval::["{context}" == "updating-cluster"]
+:within:
+endif::[]
+ifeval::["{context}" == "updating-cluster-between-minor"]
+:between:
+endif::[]
 
 [id="update-upgrading-web_{context}"]
 = Updating a cluster by using the web console
 
-If updates are available, you can update your cluster from the web console. You
-can update from one minor version to another, for example from version 4.1.17 to
-version 4.2.0, or to a patch within the minor version, for example from version
-4.1.16 to version 4.1.17.
+If updates are available, you can update your cluster from the web console.
 
 You can find information about available {product-title} advisories and updates
-link:https://access.redhat.com/downloads/content/290/ver=4.1/rhel---7/4.1.0/x86_64/product-errata[in the errata section]
+link:https://access.redhat.com/downloads/content/290/ver=4.1/rhel---7/4.2.0/x86_64/product-errata[in the errata section]
```

Comment:
I dunno about `rhel---7`; when I visit https://access.redhat.com/downloads/content/290/ver=4.1/, I'm redirected to https://access.redhat.com/downloads/content/290/ver=4.1/rhel---8/4.1.18/x86_64/product-software .

Also, I'm not wild about pinning to 4.2.0, will that only show errata fixed by 4.2.0->4.2.1?  Or will it show 4.2.0->4.2.{latest}?

And if we do pin to something in 4.2.z, you'll want `ver=4.2`, not `ver=4.1`.

Reply:
The errata page shows all of the changes for the minor release.